docs/UserGuide.md

```diff
@@ -10,23 +10,77 @@ title: User Guide
 
 ## Introduction
 
-PropertyWhiz (PropertyWhiz) is a **desktop app for managing properties and property buyers, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, PropertyWhiz can get your tasks done faster than traditional GUI apps.
+PropertyWhiz is a **desktop app for managing properties and property buyers**, tailor made for property agents. PropertyWhiz is optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). PropertyWhiz has inbuilt data analysis features like bar chart generation, as well as smart matching features to pair up your properties and buyers. 
```

Comment:
Can we use chart generation (general term) or histogram generation (specific term)?
A histogram is not a bar chart.